lib/content-services/src/lib/permission-manager/components/user-role-column/user-role-column.component.ts

```diff
@@ -38,13 +38,13 @@ import { LocalizedRolePipe } from '@alfresco/adf-core';
                 (keyup.arrowup)="$event.stopPropagation()"
             >
                 <mat-option *ngFor="let role of roles" [value]="role.role">
-                    {{ role.label | adfLocalizedRole }}
+                    {{ i18nRoleValue(role.label) | translate }}
```

Comment:
Is there a way in which we don't have to call a method in the template? Maybe using ngIf? Having a method  call in template is causing the method to be called repeatedly for each change detection cycle, which might cause performance issues in the long run

Reply:
same as other pipes I am cleaning up, they are not "pure", so zero performance optimizations... this code is just getting rid of the useless pipe, but not does any performance work. I would suggest we review the pipes later in terms of the performance, as the scope of this PR is just to remove the pipe.

Reply:
For this PR, I've changed the way the options are localised, so should improve performance.

Reply:
Yes, this looks much better now. Thanks!!